### PR TITLE
Blue Note Home: Replace "See Posts" button with query pagination

### PR DESCRIPTION
**Pull request by carolinan**
Still needs styling:
Screenshot 2023-07-25 at 10 53 30

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
Adding the class `wp-element-button` should be enough for it to have the element styling if we set up correctly on theme.json

File: blue-note/patterns/home.php [hunk 1]
```diff
@@ -32,12 +27,17 @@
...
-	<!-- /wp:heading -->
-
 	<!-- wp:query {"query":{"perPage":3,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true},"displayLayout":{"type":"flex","columns":3},"align":"wide","className":"is-style-blue-note-query-slant","layout":{"type":"default"}} -->
 	<div class="wp-block-query alignwide is-style-blue-note-query-slant">
+
+		<!-- wp:query-pagination -->
+		<!-- wp:query-pagination-next {"label":"<?php esc_html_e( 'See Posts','blue-note' ); ?>","className":"wp-element-button","fontSize":"normal"} /-->
+		<!-- /wp:query-pagination -->
+
+		<!-- wp:heading {"align":"wide","style":{"spacing":{"margin":{"top":"6.25rem"}}}} -->
```
Comment (carolinan) [inline blue-note/patterns/home.php:37]: Lets replace this margin top with a preset too once the spacing PR is merged?

**Comment by scruffian** [comment]
I don't see the button in the frontend - not sure why...

**Comment by carolinan** [comment]
> I don't see the button in the frontend - not sure why...

Is there content to page through?

**Review by MaggieCabrera — APPROVED** [review]
This works for me, I think Ben's issue is he doesn't have enough posts

**Pull request merged**